src/components/index.ts

```diff
@@ -1,3 +1,4 @@
 // eslint-disable-next-line import/no-cycle
 export { App } from './App/App';
 export { Clock } from './Clock/Clock';
+export { ErrorMessage } from './ErrorMessage/ErrorMessage'
```

Comment:
to avoid this strange pathes name component file as index.tsx

Reply:
done and in others components too